python/cuml/metrics/regression.pyx

```diff
@@ -184,9 +184,9 @@ def mean_squared_error(y_true, y_pred,
         Defines aggregating of multiple output values.
         Array-like value defines weights used to average errors.
         'raw_values' :
-            Returns a full set of errors in case of multioutput input.
+        Returns a full set of errors in case of multioutput input.
```

Comment:
I get confused about whether this should be indented or not and whether the text should be started in the line above and indented on this line. Is there global guidance on which is preferred?

Reply:
Each parameter (like 'multioutput') has a uniform level of indent for its whole description. If you try to indent any chunk further or less (like here in the sub-parameter description of raw_values), you get a warning and weird results.